packages/mui-material/src/Typography/Typography.d.ts

```diff
@@ -69,7 +69,7 @@ export interface TypographyOwnProps extends SystemProps<Theme> {
    * }
    */
   variantMapping?: Partial<
-    Record<OverridableStringUnion<Variant | 'inherit', TypographyPropsVariantOverrides>, string>
+    Record<OverridableStringUnion<Variant, TypographyPropsVariantOverrides>, string>
```

Comment:
Based on the comment below, I think best to not change this.

Also, we could rename Variant -> TypographyVariant, since this is how we export it anyway, it's strange to use a different name internally.

Reply:
> Also, we could rename Variant -> TypographyVariant, since this is how we export it anyway, it's strange to use a different name internally.

I can handle that in a separate PR to avoid including other things in this one.